src/app/tests/suites/OperationalCredentialsCluster.yaml

```diff
@@ -23,10 +23,15 @@ tests:
       command: "readAttribute"
       attribute: "SupportedFabrics"
       response:
-          value: 16
+          constraints:
+              type: uint8
+              minValue: 5
+              maxValue: 255
 
     - label: "Read number of commissioned fabrics"
       command: "readAttribute"
       attribute: "CommissionedFabrics"
       response:
-          value: 1
+          constraints:
+              type: uint8
+              minValue: 1
```

Comment:
If the test forces 1, then perhaps `value` should be checked 1. But that would mean a stateful test that assumes clear slate. This is stateless, but could falsely pass if there was 1 fabrics and we commission one, and it doesn't bump to 2

Reply:
The currents test suite is provisioning only one fabric on the device. I agree, that once we add more tests for multi-admin features, more granular checks should be made that the value is incrementing as expected.